README.md

```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`. All configured idField must be given a value.
+- identifier: either a single value for a single primaryKey field, or a literal if several fields:`{ id1: value, id2: otherValue }`. All configured primaryKey fields must be given a value.
 - data: a literal specifying the field to update
 
-### deleteOne
+### remove
 
 ```js
-import deleteOne  from 'co-postgres-queries/queries/deleteOne';
-deleteOne({ table, idFields, returnFields })(db)(ids);
+import remove  from 'co-postgres-queries/queries/remove';
+remove({ table, filterFields, returnFields })(ids);
+```
+
+Creates a query to delete entities.
+
+#### Configuration
+
+- table: the table name
+- filterFields: the fields that can be used to filter the updated rows
+- returnFields: list of fields retrieved by the query
+
+#### Parameters
+
+- filters:
+    literal specifying wanted value for given field
+    example:
+    ```js
+    {
+        field: 'value'
+    }
+    ```
+    will update only row for which field equal 'value'
+
+### removeOne
+
+```js
+import removeOne  from 'co-postgres-queries/queries/removeOne';
+removeOne({ table, primaryKey, returnFields })(identitfier);
 ```
 
 Creates a query to delete one entity.
 
 #### Configuration
 
 - table: the table name
-- idFields: list of key fields used to select the entity (default: `id`)
+- primaryKey: one or more fields representing the primary key. Accept array or single value. (default: `id`)
 - returnFields: list of fields retrieved by the query
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`
+- identitfier: either a single value for a single primaryKey field, or a literal if several fields:`{ id1: value, id2: otherValue }`. All configured primaryKey fields must be given a value.
```

Comment:
Shouldn't this be renamed as `primaryKey` and accept either an array or a value ?